CHANGELOG

```diff
@@ -1,3 +1,37 @@
+Release 119 Tycho Magnetic Anomaly Two (2020-04-30)
+---------------------------------------------------
+Cloudfront Collector: deprecate (#4319)
+Clojure Collector: deprecate (#4264)
+Spark Enrich: deprecate (#4263)
+Beam Enrich: extract into separate repo (#4282)
+Scala Stream Collector: bump to 1.0.1 (#4338)
+Scala Stream Collector: add Snowplow Bintray to resolvers (#4326)
+Scala Stream Collector: publish Docker image for stdout via Travis (#4333)
+Scala Stream Collector: fix config example (#4332)
+Scala Stream Collector: fix incompatible jackson dependencies to enable CBOR (#4266)
+EmrEtlRunner: bump to 0.37.0 (#4297)
+EmrEtlRunner: switch AMI bootstrap scripts to HTTPS (#4256)
+EmrEtlRunner: set ig.count (#4285)
+EmrEtlRunner: Catch and retry EMR connection exceptions (#4290)
+Stream Enrich: bump to 1.1.0 (#4296)
+Stream Enrich: add the possibility to override the DynamoDB endpoint (#3942)
+Stream Enrich: use info generated by BuildInfo for Processor in Source (#4335)
+Stream Enrich: remove sbt version from README (#4303)
+Stream Enrich: bump Scala Common Enrich to 1.1.0 (#4295)
+Stream Enrich: allow to download data from private S3 or GCS (#4269)
+Scala Common Enrich: bump to 1.1.0 (#4294)
+Scala Common Enrich: use recent timestamp for unit tests in WeatherEnrichmentSpec (close #4339)
```

Comment:
close?

Reply:
Good catch! This one was added after all the others following the problem with weather enrichment unit tests